### DISCUSSION TITLE
New verification method (MinecraftID)

### DISCUSSION THREAD

**Discussion by Permanently** [NamelessMC/Nameless#2152] [Ideas]
**Please describe the feature in as much detail as possible**

I tried using MCAssoc, but have had negative feedback from my community as it's a "pain in the ass". So I've left it disabled, but have still felt slightly cautious about people signing up fake Minecraft accounts to the website.

I came across this just now and thought it would be a great addition: https://minecraft.id/
By adding this as a verification process, it'll give website administrators a choice on how users can verify their accounts.

**Comment by tadhgboyle** [comment]
I think this is a great idea

**Comment by Derkades** [comment]
I'd say the msassoc system can be removed entirely without being replaced by something else. The NamelessMC spigot plugin can do the same thing with the /validate command, on your own server.

**Comment by Permanently** [comment]
> I'd say the msassoc system can be removed entirely without being replaced by something else. The NamelessMC spigot plugin can do the same thing with the /validate command, on your own server.

Nah. I have the Nameless plugin spewing errors on a few of my servers, I don't think it's exactly stable enough to have that as the only validation command right now.

**Comment by Derkades** [comment]
> Nah. I have the Nameless plugin spewing errors on a few of my servers, I don't think it's exactly stable enough to have that as the only validation command right now.

It should work just fine, have you tried the troubleshooting steps [here](https://github.com/NamelessMC/Nameless/wiki/Plugin-is-not-working)? I'm pretty sure the plugin is quite stable at the moment, besides that it throws confusing errors instead of sending a nice message when there's a configuration error or a proxy is blocking requests from the plugin to the website.
